tidb-cloud/manage-user-access.md

```diff
@@ -1,125 +1,141 @@
+
+5. Click **Confirm**.
+
+### Invite an organization member
+
+If you are the owner of an organization, you can invite organization members. Otherwise, skip this section.To invite a member to an organization, perform the following steps:
 
 1. Click the account name in the upper-right corner of the TiDB Cloud console.
 
-2. Click **Account**. The **Profile** tab is selected by default.
+2. Click **Organization Settings**. The organization settings page is displayed.
 
-3. Update the profile information, and then click **Save**.
+3. Click **User Management**, and then select the **By All Users** tab.
 
-## View the organization and project
+4. Click **Invite**.
 
-TiDB Cloud provides a hierarchical structure based on organizations and projects to facilitate the management of your TiDB cluster. In the hierarchy of organizations and projects, an organization can contain multiple projects and organization members, and a project can contain multiple clusters and project members.
+5. Enter the email address of the user to be invited, select a role for the user, and then choose a project for the user.
 
-Under this structure:
+    > **Tip:**
+    >
+    > If you want to invite multiple members at one time, you can enter multiple email addresses.
 
-- Billing occurs at the organization level, while retaining visibility of usage in each project and cluster.
+6. Click **Confirm**. Then the new user is successfully added into the user list. At the same time, an email is sent to the invited email address with a verification link.
 
-- You can view all members in your organization.
+7. After receiving this email, the user needs to click the link in the email to verify the identity, and a new page shows.
 
-- You can also view all members in your project.
+8. If the invited email address has not been signed up for a TiDB Cloud account, the user is directed to the sign-up page to create an account. If the email address has been signed up for a TiDB Cloud account, the user is directed to the sign-in page, and after sign-in, the account joins the organization automatically.
 
-To access a cluster in a project under an organization, a user must be both a member of the organization and a member of the project. Organization owners can invite users to join the project to create and manage clusters in the project.
+> **Note:**
+>
+> The verification link in the email will expire in 24 hours. If your user doesn't receive the email, click **Resend**.
 
-To view which organizations you belong to, perform these steps:
+### Remove an organization member
 
-1. Click the account name in the upper-right corner of the TiDB Cloud console.
-2. Select **Switch Organization** in the drop-down menu, and you can see all the organizations you belong to in the submenu.
+If you are the owner of an organization, you can remove organization members. Otherwise, skip this section.To remove a member from an organization, perform the following steps:
```

Comment:
```suggestion
If you are the owner of an organization, you can remove organization members. Otherwise, skip this section.

To remove a member from an organization, perform the following steps:
```